backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/starrating/BpkStarRating.kt

```diff
@@ -189,3 +199,37 @@ private enum class BpkRatingStarType {
   Half,
   Full,
 }
+
+private fun Modifier.interactiveStarRatingSemantics(
+  rating: Float,
+  maxRating: Int,
+  onRatingChanged: ((Int) -> Unit)?,
+  contentDescription: ContentDescriptionScope.(Float, Int) -> String,
+): Modifier = composed {
+  val range = 0f..maxRating.toFloat()
+  val scope = rememberContentDescriptionScope()
+  semantics(mergeDescendants = true) {
+
+    // this is needed to use volume keys
+    if (onRatingChanged != null) {
+      setProgress { targetValue ->
+        val newValue = targetValue
+          .coerceIn(range)
+        if (newValue != rating) {
+          onRatingChanged(newValue.roundToInt())
+          true
+        } else {
+          false
+        }
+      }
+    }
+    // override describing percents
+    stateDescription = scope.contentDescription(rating, maxRating)
+  }
+    .progressSemantics(
```

Comment:
as this is applied also when `onRatingChanged` is null - what's the accessibility behaviour for the static stars?

Reply:
This is now fixed!